src/views/tables/PositionsTable/PositionsActionsCell.tsx

```diff
@@ -67,30 +66,28 @@ export const PositionsActionsCell = ({
 
   return (
     <ActionsTableCell>
-      {isSlTpEnabled &&
-        !testFlags.isolatedMargin &&
```

Comment:
is it safe to remove this now before the isolated markets pr goes in?

Reply:
yup - since we just keep it in its original spot (which is no longer flagged). That being said gonna hold off on merge until isolated markets PR is in cuz this'll be way easier to rebase cc @jaredvu